documentation/content/en/books/fdp-primer/asciidoctor-primer/_index.adoc

```diff
@@ -240,8 +232,13 @@ And this would be rendered as.
 +
 extref:{ipsec-must}[IPSec-Must article]
 
+[[asciidoctor-link-macro]]
+=== The link macro
+
+This macro is not often needed. Use cases are https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/#when-to-use-the-link-macro[specified in Asciidoctor Docs].
```

Comment:
Amongst the few use cases: 

- The target must be enclosed in a passthrough to escape characters with special meaning

<https://github.com/freebsd/freebsd-doc/pull/223#issuecomment-1636760604>

> … languages with non-latin …

Would passthrough apply? 

Under **Escape and Prevent Substitutions**, [Passthroughs](https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#passthroughs) begins: 

> A passthrough is the primary mechanism by which to escape content in AsciiDoc. …

HTH